### PR TITLE
chore(site): refactor stories and test from page components

### PR DESCRIPTION
**Pull request by BrunoQuaresma**
- Update Storybook usage to use v7 API
- Remove render tests from integration tests and move them to stories

### CODE REVIEW

**Comment by BrunoQuaresma** [comment]
I did my best to rebase but it was just nuts... sorry 😢

File: site/src/components/Conditionals/ChooseOne.stories.tsx [hunk 1]
```diff
@@ -1,46 +1,71 @@
...
+  args: {
+    children: [
+      <Cond key="1" condition>
+        The first one shows.
+      </Cond>,
+      <Cond key="2" condition={false}>
+        The second one does not show.
+      </Cond>,
+      <Cond key="3">The default does not show.</Cond>,
+    ],
```
Comment (aslilac) [inline site/src/components/Conditionals/ChooseOne.stories.tsx:22]: I liked the earlier commit where these were fragments: React will perform better with those. Does it break the `ChooseOne` component or something?

**Review by aslilac — APPROVED** [review]
Seems like tons of good stuff in here! Having Chromatic compare stories will definitely be a better solution than some of these tests getting removed.

File: site/src/components/Conditionals/ChooseOne.stories.tsx (hunk 1 above)
Comment (BrunoQuaresma) [reply site/src/components/Conditionals/ChooseOne.stories.tsx:22]: Nops, it is just for the storybook. I didnt find a similar API to keep it as a fragment in the new Storybook v7 API

**Pull request merged**